packages/salesforcedx-sobjects-faux-generator/src/transformer/transformerFactory.ts

```diff
@@ -50,9 +50,7 @@ export class SObjectTransformerFactory {
       retrievers.push(new MinObjectRetriever());
       generators.push(standardGenerator);
     } else {
-      const connection = await SObjectTransformerFactory.createConnection(
-        projectPath
-      );
+      const connection = await SObjectTransformerFactory.createConnection();
```

Comment:
Can we remove project path as a parameter to this function since it is no longer used?